lib/src/dbs/iterate.rs

```diff
@@ -107,10 +107,6 @@ impl Iterable {
 						if !res.is_empty() {
 							// Get total results
 							let n = res.len();
-							// Exit when settled
-							if n == 0 {
-								break;
-							}
```

Comment:
`!res.is_empty()` covers this, so its redundant. there is a `break;` right after the `if` statement.

Reply:
Good spot!